Azure SQL Database Deployment - Output variable of Export can't be used for Import

## Environment
- Server
  Azure Pipelines

- Agent
  Hosted - VS2017

## Issue Description

Using Azure SQL Database Deployment tasks to export and import a database for integration tests, the output variable of Export task, can't be used 'as-is' for consequent Import task.
**I think the output variable is getting somewhere a starting whitespace.**
Quite sure of this, because adding a PowerShell task that calls _Trim()_ on the variable, and updates it for being used by import task, the release succeeds.

### Task logs

**Export task**
![image](https://user-images.githubusercontent.com/8156032/53257849-70c91c80-36cb-11e9-9f6f-759908af8a8d.png)


`Successfully exported database and saved it to file 'D:\a\r1\a\GeneratedOutputFiles\Grid1.bacpac'.
`
`Generated file D:\a\r1\a\GeneratedOutputFiles\Grid1.bacpac. Uploading file to the logs.
`
`Setting output variable 'SqlDeploymentOutputFile' to 'D:\a\r1\a\GeneratedOutputFiles\Grid1.bacpac'
`
`2019-02-22T16:24:43.0076175Z ##[section]Finishing: Backup live DB`

### Error logs

**Import task**
![image](https://user-images.githubusercontent.com/8156032/53257908-92c29f00-36cb-11e9-8b68-f3010dd7ddab.png)

`##[error]No files were found to deploy with search pattern D:\a\r1\a\ D:\a\r1\a\GeneratedOutputFiles\Grid1.bacpac`

--- a/Tasks/SqlAzureDacpacDeploymentV1/SqlAzureActions.ps1
+++ b/Tasks/SqlAzureDacpacDeploymentV1/SqlAzureActions.ps1
@@ -43,7 +43,7 @@ function Export-Bacpac {
     Write-Host (Get-VstsLocString -Key "SAD_GeneratedFile" -ArgumentList "$targetBacpacFilePath")
     Write-Host "##vso[task.uploadfile]$targetBacpacFilePath"
     Write-Host (Get-VstsLocString -Key "SAD_SetOutputVariable" -ArgumentList "SqlDeploymentOutputFile", $targetBacpacFilePath)
-    Write-Host "##vso[task.setVariable variable=SqlDeploymentOutputFile] $targetBacpacFilePath"
+    Write-Host "##vso[task.setVariable variable=SqlDeploymentOutputFile]$targetBacpacFilePath"
 }
 
 function Import-Bacpac {

